### PR TITLE
Deleted Yeoman BoilerPlate stuff

### PR DESCRIPTION
**Pull request by marcusp619**
Was working on my listed on on Trello here is the link https://trello.com/c/3Aeh8gsl.
I removed all the Yeoman Boilerplate.
Cleaned up the index.html 
Added jQuery and Bootstrap CDN. 

Still need to clean up all the CSS.

### CODE REVIEW

**Comment by simon-xiong-ca** [comment]
thank you!! I'd like to get this merged first so we can see our app in progress.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="letter-button" onclick="pickLetter(this)">W</div>
+        <div class="letter-button" onclick="pickLetter(this)">X</div>
+        <div class="letter-button" onclick="pickLetter(this)">Y</div>
+        <div class="letter-button" onclick="pickLetter(this)">Z</div>
+       </div>
+
+
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="scripts/styles.js"></script>
```
Comment (simon-xiong-ca) [inline index.html]: src="scripts.js"

**Comment by dsberger** [comment]
The Yeoman stuff is gone but it looks like the styling and JS has been stripped out too? May be related to Simon's comment.

**Comment by marcusp619** [comment]
The scripts.js path was corrected. It should fade properly now

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="letter-button" onclick="pickLetter(this)">W</div>
+        <div class="letter-button" onclick="pickLetter(this)">X</div>
+        <div class="letter-button" onclick="pickLetter(this)">Y</div>
+        <div class="letter-button" onclick="pickLetter(this)">Z</div>
+       </div>
+
+
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="/styles.js"></script>
```
Comment (simon-xiong-ca) [inline index.html]: change styles.js to scripts.js :)

**Comment by simon-xiong-ca** [comment]
After the jquery and scripts.js changes it should be good to go. As this is a blocker, could we merge this and iron out the details later?

File: index.html (hunk 2 above)
Comment (marcusp619) [inline index.html]: Fixed

**Comment by marcusp619** [comment]
I feel its ready for a merge let me ask Dan.

**Comment by marcusp619** [comment]
@simonxca Okay, Dan said we were good to go on this one! Merging!

**Pull request merged**